mlir/include/mlir/Conversion/FuncToEmitC/FuncToEmitCPass.h

```diff
@@ -0,0 +1,24 @@
+//===- FuncToEmitCPass.h - Func to EmitC Pass -------------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef MLIR_CONVERSION_FUNCTOEMITC_FUNCTOEMITCPASS_H
+#define MLIR_CONVERSION_FUNCTOEMITC_FUNCTOEMITCPASS_H
+
+#include <memory>
+
+namespace mlir {
+class Pass;
+
+#define GEN_PASS_DECL_FUNCTOEMITC
+#include "mlir/Conversion/Passes.h.inc"
+
+std::unique_ptr<Pass> createConvertFuncToEmitC();
```

Comment:
This shouldn't be needed?

Reply:
It seems common to most  passes to declare it in the header, but yes, it isn't needed if one uses the correct definition (`GEN_PASS_DECL_CONVERTFUNCTOEMITC` instead of `GEN_PASS_DECL_FUNCTOEMITC`). Thanks for the pointer!